packages/wonder-blocks-popover/components/popover-context.js

```diff
@@ -0,0 +1,16 @@
+// @flow
+import * as React from "react";
+
+import type {Placement} from "@khanacademy/wonder-blocks-tooltip";
+
+type ContextType = {|
+    close?: () => mixed,
+    placement?: Placement,
+|};
+
+const defaultContext: ContextType = {
+    close: undefined,
+    placement: "top",
+};
+
+export default React.createContext<ContextType>(defaultContext);
```

Comment:
Can you add some commentary here on what this is? Should help future maintenance and code spelunking.